src/Filter/FilterIn.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Filter;
+
+use Assert\Assertion;
+use Lmc\ApiFilter\Entity\Value;
+
+class FilterIn extends AbstractFilter
```

Comment:
I think it would be better to change naming to `InFilter`. `FilterIn` just don't sounds good to me. For example, controllers and services have same suffix `IndexController`, `SomeService`... What do you think?

Reply:
I dont plan any other filter atm